ValheimPlus/RPC/VPlusConfigSync.cs

```diff
@@ -18,15 +21,43 @@ public static void RPC_VPlusConfigSync(long sender, ZPackage configPkg)
                 string[] rawConfigData = File.ReadAllLines(ConfigurationExtra.ConfigIniPath);
                 List<string> cleanConfigData = new List<string>();
 
-                for (int i = 0; i < rawConfigData.Length; i++)
-                {
-                    if (rawConfigData[i].Trim().StartsWith(";") ||
-                        rawConfigData[i].Trim().StartsWith("#")) continue; //Skip comments
+                FileIniDataParser parser = new FileIniDataParser();
+                IniData configdata = parser.ReadFile(ConfigurationExtra.ConfigIniPath);
```

Comment:
A change I was planning was to load the config in once at mod init and use it throughout, rather than loading it repeatedly for sync (the mod doesn't reapply new values so if the server owner changes the config while the server is running it will be syncing mismatched values to the client). Could you make that change here since you're refactoring anyway?

Reply:
Was too lazy to serialize the config and took the easy way to use IniParser, will do.